proposals/0386-package-access-modifier.md

```diff
@@ -117,9 +118,9 @@ engine.run() // Error: cannot find `run` in scope
 
 ### Package Names
 
-Two modules belong to the same package if they were built with the same package name.  A package name must be a unique string of US-ASCII alphanumeric characters, `_`, `.`, or `-`; that is, it must match the regular expression `\A[A-Za-z0-9_.-]+\z`. It is passed to the Swift frontend via a new flag `-package-name`.  
+Two modules belong to the same package if they were built with the same package name.  A package name must be (1) a unique identity string and (2) a valid UTF-8 string starting with a letter or `_`; for example, a string such as "Προϊόν" (translation "product") must be converted to "Προι_ο_ν" to be processed. A common package name includes alphanumeric characters (starting with a letter), `-`, `_`, and `.`.
```

Comment:
After the rewrite, I'm not sure that the sentence `a string such as "Προϊόν" (translation "product") must be converted to "Προι_ο_ν"` naturally follows from the preceding two requirements anymore; the requirements just state that it is unique and UTF-8.

Reply:
updated